Makefile

```diff
@@ -170,7 +170,8 @@ htmlview: html
 
 .PHONY: htmllive
 htmllive: SPHINXBUILD = $(VENVDIR)/bin/sphinx-autobuild
-htmllive: SPHINXOPTS = --re-ignore="/\.idea/|/venv/" --open-browser --delay 0
+# Arbitrarily selected ephemeral port between 49152–65535:
```

Comment:
```suggestion
# Arbitrarily selected ephemeral port between 49152–65535
# to avoid conflicts with other processes:
```
I would also state the problem being solved.